README.md

```diff
@@ -57,24 +55,22 @@ Contributions are welcome!
 ### Guidelines for Non-Code and other Trivial Contributions
 Please keep in mind that we do not accept non-code contributions like fixing comments, typos or some other trivial fixes. Although we appreciate the extra help, managing lots of these small contributions is unfeasible, and puts extra pressure in our continuous delivery systems (running all tests, etc). Feel free to open an issue pointing to any of those errors, and we will batch them into a single change.
 
-1. [Create an issue](https://github.com/Consensys/linea-monorepo/issues)
+1. [Create an issue](https://github.com/status-im/status-network-monorepo/issues)
 > If the proposed update is non-trivial, also tag us for discussion.
-2. Submit the update as a pull request from your [fork of this repo](https://github.com/Consensys/linea-monorepo/fork), and tag us for review.
+2. Submit the update as a pull request from your [fork of this repo](https://github.com/status-im/status-network-monorepo/fork), and tag us for review.
 > Include the issue number in the pull request description and (optionally) in the branch name.
 
-Consider starting with a ["good first issue"](https://github.com/ConsenSys/linea-monorepo/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
+Consider starting with a ["good first issue"](https://github.com/status-im/status-network-monorepo/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
 
 Before contributing, ensure you're familiar with:
 
 - Our [contribution guide](docs/contribute.md)
 - Our [code of conduct](docs/code-of-conduct.md)
-- The [Besu contribution guide](https://wiki.hyperledger.org/display/BESU/Coding+Conventions), for Besu:Linea related contributions
 - Our [Security policy](docs/security.md)
 
 ### Useful links
 
-- [Linea docs](https://docs.linea.build)
-- [Linea blog](https://linea.mirror.xyz)
-- [Support](https://support.linea.build)
-- [Discord](https://discord.gg/linea)
-- [Twitter](https://twitter.com/LineaBuild)
+- [Status Network home](https://status.network)
+- [Status Network docs](https://docs.status.network)
+- [Status blog](https://status.app/blog)
```

Comment:
link to blog isn't needed here
it'll be better to add a link to the Telegram builder's community: https://t.me/statusl2